StatsdClient/StatsdClientExtensions.cs

```diff
@@ -34,7 +48,48 @@ public static void LogTiming(this IStatsd client, string name, long milliseconds
         /// <param name="value">The value for this gauge</param>
         public static void LogGauge(this IStatsd client, string name, int value)
         {
-            client.LogGaugeAsync(name, value).Wait();
+            try
+            {
+                client.LogGaugeAsync(name, value).Wait();
+            }
+            catch(AggregateException ex)
+            {
+                throw ex.InnerException;
+            }
+        }
+
+        /// <summary>
+        /// Log a gauge.
+        /// </summary>
+        /// <param name="name">The metric name</param>
+        /// <param name="value">The value for this gauge</param>
+        public static void LogGauge(this IStatsd client, string name, double value)
+        {
+            try
+            {
+                client.LogGaugeAsync(name, value).Wait();
```

Comment:
Can you revert this change, the aggregate exception should be boiled up as it can contain multiple exceptions.